index.js

```diff
@@ -477,6 +477,9 @@ module.exports = function (config, deps) {
         .end(
         function (err, res) {
           if (err != null) {
+            if (err.status !== 201) {
```

Comment:
Just to follow up on this after a video chat between myself, Chris and Jamie last night. In this PR we will keep all functionality to mimic previous behaviour, and I will suggest changes to improve error tracing in a separate PR.